### PR TITLE
separate operator case so we do not need to resolve entity

### PR DESCRIPTION
**Pull request by PetervDooren**

### CODE REVIEW

**Comment by PetervDooren** [comment]
This has been tested and it indeed resolves the hotfix installed in ed_object models

**Review by LarsJanssenTUe — APPROVED** [review]

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]
As discussed in the online meeting

File: action_server/src/action_server/actions/navigate_to.py [hunk 1]
```diff
@@ -94,11 +88,26 @@ def _configure(self, robot, config):
                 self._config_result.required_context['source-location'] = config.semantics['source-location']
             return
         # Now we can assume we know the navigation goal entity!
+        # operator
+        if semantics.target_location.id == 'operator':
+            # If we need to navigate to "me", which resolves to "operator", plant a waypoint at the current position to
+            # navigate to.
+            # TODO: learn to recognize the operator so that you know you found him later on
+            self._robot.ed.update_entity(uuid="operator", frame_stamped=self._robot.base.get_location(),
```
Comment (MatthijsBurgh) [inline action_server/src/action_server/actions/navigate_to.py:96]: Is this not done somewhere else in our software stack??

Comment (PetervDooren) [reply action_server/src/action_server/actions/navigate_to.py:96]: Perhaps, but integration of that in the action server is not the goal of this PR. This is a bug fix of the current implementation

**Comment by PetervDooren** [comment]
@MatthijsBurgh regarding the need for an attribute check for target_location.id: if this were needed this code would break on line 65 long before it reaches these changes

**Review by MatthijsBurgh — APPROVED** [review]

**Comment by MatthijsBurgh** [comment]
Thanks @PetervDooren  for fixing this.

**Pull request merged**